docker-compose.yml

```diff
@@ -1,40 +1,20 @@
-version: '2.2'
-
+volumes:
+  postgres_data_local: {}
 services:
-  database:
-    image: "postgres:9.6"
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
-      interval: 10s
-      timeout: 5s
-      retries: 5
+  db:
+    image: postgres:14
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data_local:/var/lib/postgresql/data
 
   app:
-    links:
-    - database
-    depends_on:
-      database:
-        condition: service_healthy
+    ports: 
+      - "3000:3000"
+    env_file:
+      - .env
     build: .
-    environment:
-      RACK_ENV:
-      RAILS_ENV:
-      DATABASE_URL:
-      SECRET_KEY_BASE:
-      ADMIN_EMAILS:
-
-      # SENDGRID or SES
-      SMTP_PROVIDER:
-
-      # Sendgrid
-      SENDGRID_USERNAME:
-      SENDGRID_PASSWORD:
-
-      # SES
```

Comment:
📝 we'll probably need to bring SES back once we start trying to get this on AWS but agree with removing it here for now